### DISCUSSION TITLE
export_model.py出来的英文模型识别模型不能运行？

### DISCUSSION THREAD

**Discussion by wuqiangch** [PaddlePaddle/PaddleOCR#12799] [Q&A]
在官方下载的en_PP-OCRv3_rec_infer模型在C++里面可以运行，在官方下载en_PP-OCRv3_rec_train或者用这个模型微调出来的模型，再用export_model.py无法使用，对比发现inference.pdiparams这个文件是一样，但是inference.pdmodel这个文件大小不一样。官网下载的这个文件要小于我自己export出来的文件，这个是什么原因？提示错误：没有为 paddle_inference.dll 加载的符号文件，是不是版本不兼容呢？
Linux下面训练和导出模型使用的paddle版本：2.5.1，paddleocr版本：2.7
windows用的c++库版本： avx_mkl_cuda11.6_cudnn8.4_avx_mkl-trt8.4.1.5 | paddle_inference.zip（2.3.2） | MSVC 2019 | 8.4 | 11.6
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/7983007/4d76d56c-9131-452c-872b-31746546f1c2)

**Comment by shiyutang** [comment]
issuebot提供了相关解答，请查看是否满足要求：https://github.com/shiyutang/Acceleratedcpp/issues/5

**Comment by balcklive** [comment]
same issue

**Comment by DouLiu95** [comment]
> issuebot提供了相关解答，请查看是否满足要求：[shiyutang/Acceleratedcpp#5](https://github.com/shiyutang/Acceleratedcpp/issues/5)

你好，我在colab环境（paddlepaddle=2.5.1 cuda=12.0）里自训练了layoutxlm模型，导出的推理模型也可以正常工作。之后想部署在docker container的环境里（paddlepaddle=2.3.2 noavx cpu）但是模型预测时会报错：
```
ValueError: (InvalidArgument) Failed to parse program_desc from binary string.
  [Hint: Expected desc_.ParseFromString(binary_str) == true, but received desc_.ParseFromString(binary_str):0 != true:1.] (at /paddle/paddle/fluid/framework/program_desc.cc:143)
```
这个问题的原因也是训练和部署的环境不匹配吗？

但是另一个疑问是，我在两个环境里都可以使用教程里提供的inference模型推理，为什么官方的inference在不同环境里可以预测但是我自己export的inference model不行呢？
